apmpackage/apm/data_stream/traces/elasticsearch/ingest_pipeline/default.yml

```diff
@@ -15,6 +15,8 @@ processors:
       name: client_geoip
   - pipeline:
       name: event_duration
+  - pipeline:
+      name: event_success_count
```

Comment:
Seeing as this one only applies to trace events, should we define it inline here?

Reply:
it was put into a named pipeline because I didn't know whether this should be used in apm_data_stream_migration as well. If it only applies to traces default, then yes it is better to inline it

Reply:
done